host_vars/aarnet-user-nfs.usegalaxy.org.au.yml

```diff
@@ -1,14 +1,18 @@
 attached_volumes:
   - device: /dev/vdb
-    path: /mnt
+    path: /mnt/perth
     fstype: ext4
   - device: /dev/vdc
     path: /user-data-0
     fstype: ext4
+  - device: /dev/vdd
+    path: /mnt/aarnet/user-data-3
```

Comment:
the role needs fstype to be defined or it will skip it

is this /mnt/aarnet/user-data-3 or /mnt/aarnet?